adyenofficial.php

```diff
@@ -1529,7 +1561,9 @@ private function getOneClickPaymentMethods(array $paymentMethods)
                 $smartyVariables = array(
                     'storedPaymentApiId' => $storedPayment['id'],
                     'name' => $storedPayment['name'],
-                    'number' => $storedPayment['lastFour']
+                    'logoBrand' => $storedPayment['brand'],
+                    'number' => $storedPayment['lastFour'],
+                    'removeCollapse' => \Configuration::get('ADYEN_REMOVE_PAYMENT_DISPLAY_COLLAPSE')
```

Comment:
In this case if the configuration cannot be found then there is no default value.
A quick solution can be to get the configuration before assigning the value to the smartyVariables array and check if the value is 1 (or true I'm not sure which one you are going to store in the db because of the `is_bool` field) otherwise you can assign 0 (or false) to it so you can be sure what you pass to the frontend is something you know for sure.

Longer solution: PW-1839 to have default value for both 1.6 and 1.7 (A branch is already prepared for it I'm not sure if anything is missing from the feature branch)

Reply:
Change committed